src/main/java/org/cactoos/text/Newline.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2020 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.text;
+
+import org.cactoos.Text;
+
+/**
+ * Returns a text that is the default system line separator.
+ *
+ * <p>There is no thread-safety guarantee.
+ * @since 0.50
+ */
+public final class Newline extends TextEnvelope {
+    /**
+     * Ctor.
+     */
+    public Newline() {
+        this("");
+    }
+
+    /**
+     * Ctor.
+     * @param text The text
+     */
+    public Newline(final String text) {
+        this(new TextOf(text));
+    }
+
+    /**
+     * Ctor.
+     * @param text The text
+     */
+    public Newline(final Text text) {
+        super(new Concatenated(new FormattedText("%n"), text));
```

Comment:
@dearrudam I don't understand why we could need any of the last two constructors: we need only one `Newline()` delegating to `FormattedText("%n")`, no? What would be the use case for the others?

Reply:
> @dearrudam I don't understand why we could need any of the last two constructors: we need only one `Newline()` delegating to `FormattedText("%n")`, no? What would be the use case for the others?

Well, as @andreoss asked to make the `Newline` a `TextEnvelope` then I thought that such one needs to "envelope" other components as needed. Do you think that it's better to change it to be a simple `TextEnvelope` with one only constructor like `Newline()`?

Reply:
@dearrudam yes, @andreoss request was correct, but it's not needed to introduce unnecessary complexity. Actually envelopes are often used exactly like this, with only one constructor compositing other objects to build the desired behaviour.

`Newline` is implemented with `TextEnvelope`, but `Newline` from the point of view of its users is NOT an envelope.

Reply:
Great! Got it! I'm changing it right away! Thanks!